Create SliderBanner component

Taps73r opened the pull request:
Create SliderBanner component:
1) SliderBanner with react-slick and scss styles
2) Response for mobile and other devices.
3) SampleArrow component for slider.
Update package.json:
Added new dependencies:
"@types/react-slick", "react-slick", "slick-carousel" for slider working.

Taps73r committed:
Create SliderBanner component:
1) SliderBanner with react-slick and scss styles
2) Response for mobile and other devices.
3) SampleArrow component for slider.
Update package.json:
Added new dependencies:
"@types/react-slick", "react-slick", "slick-carousel" for slider working.

Kumala3 commented on frontend/src/components/SliderButtons/SampleNextArrow.tsx:
### Great naming for the arrow components!
#### Change "SampleNextArrow" function name and the component and do the same with "PrevArrow" and don't forget about styles component name too!

Taps73r committed:
Revised version of the pull request:
1) Removed the .gitkeep file.
2) Renamed classes as per the instructions in the comments.
3) Moved SVG files to the images folder.
4) Renamed the left and right arrow names according to the comments.

Taps73r commented:
Can you review all the changes in the new commit? All the modifications made are described there.

Kumala3 commented on frontend/package.json:
### Update dependencies up to date with the develop branch please!

Kumala3 commented on frontend/src/components/SliderButtons/NextArrow.tsx:
Make empty line between interface and imports as you do everywhere other.

Kumala3 commented on frontend/src/components/SliderButtons/SliderButton.scss:
Make empty lines between styles for more readable, please.

Kumala3 commented on frontend/src/components/SliderBanner/SliderBanner.tsx:
Update your SliderBanner up to date with develop branch. I mean change string param to Banner from types and make only one empty line between imports and interface not two.

Taps73r committed:
Format code.

Taps73r committed:
Merge branch 'develop' into feature/component/sliderbanner.
Resolve conflicts in package.json

Taps73r committed:
Change bannerPhotos to bannerData.
Change interface bannerData to Banner[].
In tag img change src to img.season_image_url

Taps73r commented:
Check new changes in PR.
Merge develop branch with feature/component/sliderbanner branch.
Resolve conflict in package.json.
Change interface in SliderBanner.
Change name for props in SliderBanner.

Kumala3 reviewed (changes requested):
Delete unnecessary images from the project.
Don't forget to add a newline at the end of code, everywhere where it isn't.

Taps73r committed:
Add newline at the end of code in:
Next and Prev arrow component;
WatchButton.tsx & .scss;
SliderButton.scss;

Taps73r committed:
Delete test pictures

Taps73r commented:
Delete test pictures from the project.
Add newline at the end

Taps73r committed:
Add alt attribute for arrow components and SliderBanner

Taps73r committed:
Create better responsive styles, change slider responsive settings and interface

Taps73r commented:
**Ch### anges in the SliderBanner.scss file:**

- Made the height and width of the slide dependent on the device screen width.
- Reduced the spacing between slides.
- Increased the value of transform: scale.
### 
**Changes in the SliderBanner.tsx file:**

- Modified responsive settings by adding onFocus for mobile phones and tablets, removed unnecessary settings.
- Adjusted the interface according to the new responsive settings.

Taps73r committed:
Add space to .scss files, fix background for buttons and buttons size

Taps73r committed:
Change div to button, and small fix in interface

Taps73r commented:
### .scss flies change:
add background for buttons,
delete border for buttons and change button size,
in all .scss files add 4 space for better reading.

### NextArrow.tsx PrevArrow.tsx change:
change div to button and fix interface

Kumala3 reviewed (approved):
### Great job, keep it up! PR include everything what was described in #17.
PR can be merged with the develop branch.
